### ISSUE TITLE
no funciona para crear suscripciones

### ISSUE THREAD

**Issue by matiazar** [Gazer/px-flutter#38]
Buenas tardes. como se podra implementar la funcionalidad para poder crear suscripciones?

tengo entendido que el proceso es el mismo, salvo que cambia el endpoint a donde direccionar la llamada.

gracias!

**Comment by Gazer** [comment]
No, hasta donde tengo entendido las librerías nativas no soportan
suscripciones.

**Comment by matiazar** [comment]
que c... la libreria obviamente. tu laburo impecable.
voy a pensar una alternativa.. quizas poner un par de botones, de compra un mes del servicio y compra un año del servicio.
y listo. manejar en mi backend la fecha de inicio y fin.
